docs/codeql/codeql-for-visual-studio-code/exploring-the-structure-of-your-source-code.rst

```diff
@@ -28,7 +28,7 @@ Viewing the abstract syntax tree of a source file
    
    .. pull-quote:: Note
 
-      If you don't have an appropriate ``printAST.ql`` query in your workspace, the **CodeQL: View AST** command won't work. To fix this, you can update your copy of the `CodeQL <https://github.com/github/codeql>`__ repository (or `CodeQL for Go <https://github.com/github/codeql-go>`__ repository) from ``main``. If you do this, you may need to upgrade your databases. Also, query caches may be discarded and your next query runs could be slower.
+      If you don't have an appropriate ``printAST.ql`` query in your workspace, the **CodeQL: View AST** command won't work. To fix this, you can update your copy of the `CodeQL <https://github.com/github/codeql>`__ repository from ``main``. If you do this, you may need to upgrade your databases. Also, query caches may be discarded and your next query runs could be slower.
```

Comment:
I'm wondering how long it is since we added the `printAST.ql` query. It's possible that we could delete this note.